README.md

```diff
@@ -158,6 +158,6 @@ If you like this project please show support by staring ⭐️.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE.txt) licensed.
+This project is [MIT](/LICENSE.txt) licensed.
```

Comment:
- [ ] The **MIT** link under the **License** section in your README file link to a **404** page. Kindly check it and make sure all the links in your README file work as intended